src/lib/DefaultRuleManager.ts

```diff
@@ -62,7 +62,7 @@ export class DefaultRuleManager implements RuleManager {
 		return this.catalog.getRulesMatchingFilters(filters);
 	}
 
-	async runRulesMatchingCriteria(filters: RuleFilter[], targets: string[], format: OUTPUT_FORMAT, engineOptions: Map<string, string>): Promise<RecombinedRuleResults> {
+	async runRulesMatchingCriteria(filters: RuleFilter[], targets: string[], outputOptions: OUTPUT_OPTIONS, engineOptions: Map<string, string>): Promise<RecombinedRuleResults> {
```

Comment:
In order to know if the user wants normalized severity or not, outputOptions is passed instead which contains the format and a normalize severity boolean